editor/import/resource_importer_scene.cpp

```diff
@@ -1381,29 +1381,26 @@ Error ResourceImporterScene::import(const String &p_source_file, const String &p
 		Map<Ref<ArrayMesh>, Transform> meshes;
 		_find_meshes(scene, meshes);
 
-		if (light_bake_mode == 2) {
+		float texel_size = p_options["meshes/lightmap_texel_size"];
```

Comment:
The `|| generate LOD` comment above should be removed, otherwise one might attempt to reenable it and this `light_bake_mode == 2` specific code would become a bug.